contact.html

```diff
@@ -4,8 +4,82 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
+    <header>
+        <div>
+            Alex Reber's Site!
+        </div>
+    </header>
+    <nav>
+        <div>
+            <a href="index.html">Home</a> 
+            <a href="contact.html">Contact</a> 
+            <a href="gallery.html">Gallery</a> 
+        </div>
+    </nav>
+    <div class="container">
+        <div style="padding-top: 5%; padding-bottom: 5%">
```

Comment:
CSS (Cascading Style Sheets) are intended to encapsulate the "presentation" (styles) from the "content" (html).  This improves the cohesion of both, allowing for change in one without impacting the other.  With a clean separation, it also avoids the situation in which someone adds a style, then is frustrated it results in _**no**_ change due to a style which has been applied in the HTML.  Consider moving all styling to the CSS files, and removing them from the HTML.